core/web/resolver/vrf.go

```diff
@@ -0,0 +1,139 @@
+package resolver
+
+import (
+	"github.com/graph-gophers/graphql-go"
+
+	"github.com/smartcontractkit/chainlink/core/services/keystore/keys/vrfkey"
+)
+
+type VRFKeyResolver struct {
+	key vrfkey.KeyV2
+}
+
+func NewVRFKeyResolver(key vrfkey.KeyV2) VRFKeyResolver {
+	return VRFKeyResolver{key}
+}
+
+// ID returns the ID of the VRF key, which is the public key.
+func (k VRFKeyResolver) ID() graphql.ID {
+	return graphql.ID(k.key.PublicKey.String())
```

Comment:
That's what I thought to do but I was mirroring the VRF key REST API where we set the ID to be the public key (https://github.com/smartcontractkit/chainlink/blob/f5f0c97e5cae7cd79aa80fa602d8710de8e1e76b/core/web/presenters/vrf_key.go#L25-L30). I'm happy to change it to use `ID()` though, which is probably the same since it encodes the public key in hexadecimal.